src/background.js

```diff
@@ -93,21 +94,40 @@ function init() {
 
   getMetricsUUID().then(clientUUID => {
     Metrics.init(clientUUID);
-    return getAlarms();
+    return getNextOpenAlarms();
```

Comment:
So, I did it that way so that we wouldn't register the window listener if we didn't need to, but it's probably not that expensive, so I'll remove it.